docs/user/ocean-simulation.rst

```diff
@@ -427,3 +427,31 @@ The *Crest/Inputs/Flow/Add Flow Map* shader writes a flow texture into the syste
 It assumes the x component of the flow velocity is packed into 0-1 range in the red channel, and the z component of the velocity is packed into 0-1 range in the green channel.
 The shader reads the values, subtracts 0.5, and multiplies them by the provided scale value on the shader.
 The process of adding ocean inputs is demonstrated in :numref:`adding-inputs-video`.
+
+.. _albedo-section:
+
+Albedo
+------
+
+.. admonition:: Preview
+
+   This feature is currently in preview.
+
+Overview
+^^^^^^^^
+
+The Albedo feature allows a colour layer to be composited on top of the water surface.
+This is useful for projecting colour onto the surface.
+
+This is somewhat similar to decals, except the colour only affects the water.
+
+HDRP has a Decal feature that works with the water, and the effect is more configurable and may be preferred over this feature. `[HDRP]`
```

Comment:
Removed label

I figured this might be a chance to consolidte in the decal section and remove it from the Other Featurs which is a bit of a mixed bag catch all section. I've merged in the delta and removed it.

Let me know if this looks ok

Reply:
Sure. Looks good.